packages/tenant-process/test/addCertifiedAttribute.test.ts

```diff
@@ -179,7 +179,7 @@ describe("addCertifiedAttribute", async () => {
     await addOneAttribute(attribute);
     await addOneTenant(tenantWithCertifiedAttribute);
     await addOneTenant(requesterTenant);
-    const returnedTenant = await tenantService.addCertifiedAttribute(
+    const { data: returnedTenant } = await tenantService.addCertifiedAttribute(
```

Comment:
As done for attributes/delegations, I would update this test to test the entire result object including metadata

See https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-2ab69d62d3cac341a38dac792805f097aee52ca413a8806d8b976398bf47d1afR82

Reply:
Why not do the same as in other tests, also in this test? Testing the entire returned value including metadata